### DISCUSSION TITLE
Relationship-Field at import

### DISCUSSION THREAD

**Discussion by markusthiel** [Budibase/budibase#12539] [Help] [answered]
I have linked two tables together using a relationship field. Now I have tried to update the table using the CSV import plugin. Unfortunately, it doesn't work with the relationship fields. There, for example, I have linked names from another table. I simply entered these names in the hope that they would be recognized. What can I do to capture and adopt the relation?

**Comment by melohagan** [comment, marked as answer]
Hey @markusthiel 

You can do this with an automation as outlined here: https://github.com/Budibase/budibase/issues/10899
